scss/_global.scss

```diff
@@ -136,6 +137,11 @@ $global-color-pick-contrast-tolerance: 0 !default;
 @mixin foundation-global-styles {
   @include -zf-normalize;
 
+  // Add the search input cancel button in Chrome/Safari again
+  [type="search"]::-webkit-search-cancel-button {
+    -webkit-appearance: searchfield-cancel-button;
+  }
+
```

Comment:
> ...or overwrite this in our global styles

Sorry, I had in mind a cleaner version of https://github.com/zurb/foundation-sites/issues/11015#issuecomment-373959867 to resolve this issue, so this overwrite would not be required. I'll take care of it.